templates/templates/preview/test.template

```diff
@@ -0,0 +1,117 @@
+{% for model in manifest.models %}
```

Comment:
- ditto on location of test template
- ditto on naming convention

Reply:
removed